### PR TITLE
Add color to Text

### PR DESCRIPTION
**Pull request by Bidjii**

### CODE REVIEW

File: src/text/index.tsx [hunk 1]
```diff
@@ -47,19 +48,25 @@ const Text = ({
...
   let content = children
   if (typeof children === 'string' && newlineToBr) {
     content = replaceNewLineWithBR(children)
   }
 
   return (
-    <Tag className={cc([baseClassName, displayClassName, className])}>
+    <Tag
+      className={cc([baseClassName, displayClassName, className])}
+      style={cssColorRegex.test(textColor) ? { color: textColor } : null}
```
Comment (pith) [inline src/text/index.tsx]: You could print a warning if the value doesn't match

File: src/text/story.tsx [hunk 2]
```diff
@@ -4,17 +4,29 @@ import { storiesOf } from '@storybook/react'
 import { withInfo } from '@storybook/addon-info'
 import { selectV2, text, withKnobs, boolean } from '@storybook/addon-knobs'
 
+import { color } from '_utils/branding'
 import Text, { TextTagType, TextDisplayType } from 'text'
 
 const stories = storiesOf('Text', module)
 stories.addDecorator(withKnobs)
 
+const c = Object.keys(color).reduce(
```
Comment (pith) [inline src/text/story.tsx]: To remove ?

Comment (pith) [reply src/text/story.tsx]: Ah ok it's used above. Can you change the name to make it more explicit ?

File: src/text/story.tsx [hunk 3]
```diff
@@ -4,17 +4,29 @@ import { storiesOf } from '@storybook/react'
...
+  (acc, key) => ({
+    ...acc,
+    [key]: color[key],
+  }),
+  {},
+)
+
+console.log(c)
```
Comment (pith) [inline src/text/story.tsx]: You can remove it

File: src/itinerary/__snapshots__/index.unit.tsx.snap [hunk 4]
```diff
@@ -16,6 +16,7 @@ exports[`Itinerary component Should display proximity from departure point 1`] =
       >
         <span
           className="kirk-text kirk-text-titleStrong"
+          style={null}
```
Comment (pith) [inline src/itinerary/__snapshots__/index.unit.tsx.snap]: 😬 would be better if we could avoid this in the snapshot

File: src/text/index.tsx [hunk 5]
```diff
@@ -47,19 +48,25 @@ const Text = ({
...
   display = TextDisplayType.BODY,
   tag = TextTagType.SPAN,
+  textColor,
   newlineToBr = true,
 }: TextProps) => {
   const baseClassName = 'kirk-text'
   const displayClassName = `${baseClassName}-${display}`
   const Tag = tag
 
+  const cssColorRegex = /^#([0-9a-f]{3}|[0-9a-f]{6})$/i
```
Comment (pith) [inline src/text/index.tsx]: Can you add a test with valid / invalid colors ?

Comment (Bidjii) [reply src/text/index.tsx]: Done

File: src/text/story.tsx (hunk 2 above)
Comment (pith) [reply src/text/story.tsx]: We agree that this code doesn't do anything ... Or I didn't get it

Comment (Tuxosaurus) [reply src/text/story.tsx]: Oh yes... the result is... `color`

Comment (Tuxosaurus) [reply src/text/story.tsx]: I commited the change

File: src/text/index.tsx (hunk 1 above)
Comment (wakooka) [reply src/text/index.tsx]: You mean a `console.warn` or `console.error`?

Comment (Tuxosaurus) [reply src/text/index.tsx]: hey @pith any further opinion on this?

**Pull request merged**